packages/action-menu/src/action-menu.css

```diff
@@ -10,15 +10,23 @@ OF ANY KIND, either express or implied. See the License for the specific languag
 governing permissions and limitations under the License.
 */
 
+:host {
+    min-width: var(--spectrum-global-dimension-400);
+}
+
+:host([quiet]) {
+    min-width: var(--spectrum-global-dimension-400);
+}
+
 #button {
     width: auto;
 }
 
-.icon {
-    /* Sizing for .icon is needed because we are using a raw <svg> due to https://github.com/adobe/spectrum-web-components/issues/155 */
-    width: 18px;
-    height: 18px;
+::slotted([slot='icon']) {
+    flex-shrink: 0;
+}
 
+.icon {
```

Comment:
This is because `::slotted()` can only point to content in the light DOM. It _seems_ like it should point to content in a slot, but that's not really what it does. Would be interesting to bring this up at the w3c...

Reply:
yeah -- having some way to target both would be... nice